.github/workflows/preview.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Any Commit
+on:
+    pull_request:
+        types: [opened, synchronize, reopened]
+    push:
+        branches:
+            - master
```

Comment:
How much utility is there in doing a publish on master? It wouldn't be any different from the preview generated in the PR (assuming the PR is up to date).

Reply:
Not much, it just would allow something like `https://pkg.pr.new/salesforce/lwc@master`. I'll remove for now.